packages/@coorpacademy-components/src/organism/sidebar/index.js

```diff
@@ -75,15 +77,28 @@ export const LinkItem = props => {
   );
 };
 
+export const TitleItem = props => {
+  return (
+    <ul data-name={props.name || `item-title-${props.index}`} className={style.titleItem}>
+      <li className={style.infoItemTitle}>{props.title}</li>
+    </ul>
+  );
+};
+
 export const InfoItem = props => {
+  const handleOnClick = e => {
```

Comment:
Not a fan of these handlers defined inside the render method, but since this one  is just a simple props.onClick, go for it!

Reply:
I do agree, not fan neither, I just _"reused"_ what was made on `LinkItem`.

If you have a better idea in mind @Fausto95  tell me :)